### DISCUSSION TITLE
Remove Sitelinks Search Box from structured data - already deprecated(removed) from 2024/11/21

### DISCUSSION THREAD

**Discussion by ShaiMagal** [PrestaShop/PrestaShop#37673] [Ideas]
### Prerequisites

- [x] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [x] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Deprecated by google
https://developers.google.com/search/blog/2024/10/sitelinks-search-box
`It's been over ten years since we initially announced the [sitelinks search box in Google Search](https://developers.google.com/search/blog/2014/09/improved-sitelinks-search-box), and over time, we've noticed that usage has dropped. With that, and to help simplify the search results, we'll be removing this visual element starting on November 21, 2024.`

### Expected behavior

Remove this, useless code now.

### Steps to reproduce

...

### PrestaShop version(s) where the bug happened

9.0.0

### PHP version(s) where the bug happened

8.3

### Your company or customer's name goes here (if applicable).

https://www.openservis.cz/

**Comment by matks** [comment]
Hi @ShaiMagal, @florine2623 has been checking a regular `develop` store and she did not find any Sitelinks attribute in the structured data.

Also I found an issue asking to implement it https://github.com/PrestaShop/PrestaShop/issues/16767 and I think it was never carried out.

**Comment by ShaiMagal** [comment]
@matks check for "SearchAction" in file "head-jsonld.tpl":
https://github.com/PrestaShop/classic-theme/blob/develop/templates/_partials/microdata/head-jsonld.tpl
https://github.com/PrestaShop/hummingbird/blob/develop/templates/_partials/microdata/head-jsonld.tpl

**Comment by matks** [comment]
@ShaiMagal Ah I was searching for "Sitelinks" I did not search for "SearchAction"
